x-pack/plugins/cases/server/client/cases/update.ts

```diff
@@ -386,6 +425,8 @@ export const update = async (
 
     throwIfUpdateOwner(casesToUpdate);
     throwIfTitleIsInvalid(casesToUpdate);
+    throwIfCategoryLengthIsInvalid(casesToUpdate);
+    throwIfCategoryIsInvalidString(casesToUpdate);
```

Comment:
nit: Let's combine them into one function like in the create case route.

Reply:
If you don't mind i'd leave this one, the assignees' validation is also done in different functions here